RNUnifiedContacts/RNUnifiedContacts.swift

```diff
@@ -272,6 +349,37 @@ class RNUnifiedContacts: NSObject {
 
     return emailAddresses
   }
+  
+  func transformNumber(number: NSDictionary) -> CNLabeledValue {
+    var label = String()
+    if (number["label"] as! String == "home") {
```

Comment:
@homanp Oh interesting. Good find! I just checked the iOS Contacts app and forces you to select a label for phone numbers and email. But that's a UI concern, not ours.

I think you're right. Allow it to handle a `null` value instead of defaulting to one. 👍